src/components/Header.jsx

```diff
@@ -6,7 +6,7 @@ import { compose } from 'redux';
 import { connect } from 'react-redux';
 import cs from 'classnames';
 import idx from 'idx';
-
+import logo from '../../public/image/ComfyApe.png'
```

Comment:
This seems to add ~4.71 KB (gzipped) to the main JS file:

![image](https://user-images.githubusercontent.com/733364/57339388-641b6880-70ff-11e9-81ec-6aace68065c4.png)

could we instead do something like `<img src='/image/ComfyApe.jpg' />`?